src/ontology/pathgo-edit.owl

```diff
@@ -3323,6 +3323,26 @@ Concept covered by GO; GO:0042981</skos:editorialNote>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/pathgo/PATHGO_0000350 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000350">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000183"/>
+        <pathgo:IAO_0000115>A gene product that modulates the host cell cytoskeleton by mediating microtubule rearrangement.</pathgo:IAO_0000115>
```

Comment:
definition should read "mediating host actin rearrangement"

Reply:
Fixed!